docs/development.md

```diff
@@ -258,3 +258,133 @@ The LoA required for RA is configured in Stepup-RA's `parameters.yml` and throug
 [saml-chrome-panel]: https://chrome.google.com/webstore/detail/saml-chrome-panel/paijfdbeoenhembfhkhllainmocckace
 [firefox-saml-tracer]: https://addons.mozilla.org/en-US/firefox/addon/saml-tracer/
 [middleware-manager]: https://github.com/OpenConext/Stepup-Middleware/tree/e1c7017f4211728157ecd49394e870858c9789c8#management-api
+
+## Howto: configure a dummy GSSP
+
+This section describes how the [GSSP example application](https://github.com/OpenConext/Stepup-gssp-example/) can be used in Stepup to test the GSSP mechanism. Below configuration allows registration and authentication on the development environment using Tiqr as second factor, using the GSSP example application instead of a real tiqr implementation.
+
+First, install, configure and run the example application. You can do that in the VM, but that's not required. Doing it on your host machine with only PHP installed might be easier.
+
+    git clone https://github.com/OpenConext/Stepup-gssp-example.git
+    cd Stepup-gssp-example
+    composer install
+
+Now configure gateway as the remote SP in `app/config/parameters.yml`:
+
+    saml_remote_sp_entity_id: 'https://gw-dev.stepup.coin.surf.net/app_dev.php/gssp/tiqr/metadata'
+    saml_remote_sp_sso_url: 'https://gw-dev.stepup.coin.surf.net/app_dev.php/gssp/tiqr/single-sign-on'
+    saml_remote_sp_certificate: '%kernel.root_dir%/../vendor/surfnet/stepup-saml-bundle/src/Resources/keys/development_publickey.cer'
+    saml_remote_sp_acs: 'https://gw-dev.stepup.coin.surf.net/app_dev.php/gssp/tiqr/consume-assertion'
+
+Finally, compile the assets and run the example application:
+
+    composer encore dev
+    cd web
+    php -S localhost:1234
+
+Verify the GSSP example application is running without error on http://localhost:1234.
+
+Now configure `Stepup-Gateway/app/config/samlstepupproviders_parameters.yml`:
```

Comment:
Did you forget to add instructions dictating the addition of `gssp_allowed_sps` entries?

Reply:
Thanks, I've included your feedback.